CentralManagement/ChocoCCM Command Reference/Set-CCMGroup.md

```diff
@@ -0,0 +1,89 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Set-CCMGroup
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
+
+## SYNTAX
+
+```
+Set-CCMGroup [[-Group] <String>] [[-NewName] <String>] [[-NewDescription] <String>] [<CommonParameters>]
+```

```

Comment:
We should allow targeting groups by ID as well here. 

I'd also just have `-NewName` be `-Name` and the same for `-Description`. The `-New` prefix isn't adding any value here IMO.